spyder/widgets/tests/test_status.py

```diff
@@ -62,53 +65,70 @@ def test_status_bar_widget_signal(status_bar, qtbot):
     assert w.get_icon() == 'icon'
 
 
-def test_status_bar_conda_status(status_bar, qtbot, mocker):
-    mocker.patch.object(utils, 'is_anaconda', return_value=True)
+def test_status_bar_conda_interpreter_status(status_bar, qtbot, mocker):
+    # We patch where the method is used not where it is imported from
```

Comment:
@mrclary this is why the mocking was not working, although not intuitive at first, we need to patch the method where we use it, not where it comes from.

Reply:
Aahh, yes, of course. I'll fix that...

Reply:
Already did it for you! :-)